File: ClockAngle/java/Angle.java
public class Angle {
    private double angle;

    public Angle(double requiredAngle) {
        angle = requiredAngle;
    }

    public Angle() {
        angle = 0;
    }

    public void convertToNormalAngle() {
        angle = Math.abs(angle);
        while (angle >= 360)
            angle -= 360;

        if (angle > 180)
            angle = 360 - angle;
    }

    public double getAngle() {
        return angle;
    }

    @Override
    public String toString() {
        return "" + angle + " degree";
    }
}

File: ClockAngle/python/angle.py
class Angle():
    def __init__(self, angle = 0.0):
        self.angle = angle

    def convert_to_normal_angle(self):
        self.angle = abs(self.angle)
        while self.angle >= 360:
            self.angle -= 360

        if self.angle > 180:
            self.angle = 360 - self.angle

    def __str__(self):
        return str(self.angle) + " degree"
